### PR TITLE
Trend Micro CVE 04192021

### PR DESCRIPTION
**Pull request by jpattrendmicro**
Modify (add new product versions and links) to CVE-2021-24556 through 24559. Add new 28648.

### CODE REVIEW

File: 2020/28xxx/CVE-2021-28648.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
```
Comment (cve-team) [inline 2020/28xxx/CVE-2021-28648.json:1]: Expected location (2021/28xxx/CVE-2021-28648.json) of file does not match actual location (2020/28xxx/CVE-2021-28648.json)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by jpattrendmicro** [comment]
Sorry, committed to 2020!  Please reject request and I'll resubmit.

**Comment by jpattrendmicro** [comment]
Cancelled the pull request and will resubmit.

Best Regards,
Jonn Perez

**Pull request closed without merging**